controllers/users-controller.js

```diff
@@ -0,0 +1,10 @@
+const {selectAllUsers} = require("../models/users-model.js");
+const {} = require("../utils/utils.js")
+
+exports.getAllUsers = (req, res, next) => {
+    selectAllUsers()
+    .then((usersArray) => {
+        res.status(200).send({ allUsers: usersArray })
+    })
+    .catch(err)

```

Comment:
This catch block needs some code to handle the error. Either .catch(err) {next(err)} or just catch(next).

Both will do the same thing; it's just that the second is refactored. Whatever function you put in the catch brackets will automatically be invoked with error by default, but let me know if you have any questions 👍